File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+---
+title: Convert Markdown to Word document in C# | DocIO | Syncfusion
+description: Convert Markdown to Word document in C# using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies
```

Comment:
Description: Convert Markdown to Word document in C# using the Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies.